gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -384,4 +391,9 @@ private long getExpiry() {
     }
     return expiry;
   }
+
+  private String getTopologyName() {
+    return (String) context.getAttribute("org.apache.knox.gateway.gateway.cluster");
```

Comment:
nit: Would be good to make this a constant. I ran into this looking into something else. It was NOT clear that this attribute was the topology name.

Reply:
This is why I put it in a method that makes it more clear (aside from the obvious benefit of re-use). I can create a constant for it. Are you suggesting a constant in this class, or somewhere else for broader consumption?

Reply:
Broader consumption - doesn't have to be as part of this change. The string `"org.apache.knox.gateway.gateway.cluster"` is being used to insert the topology value into the context.

Reply:
+1